meson.build

```diff
@@ -1,4 +1,4 @@
-project('gnome-shell', 'c',
+project('gnome-shell', ['c', 'cpp'],
```

Comment:
Also this commit should be re-authored to Sam.

Reply:
done